force-app/main/default/lwc/weatherComponent/__tests__/weatherComponent.test.js

```diff
@@ -0,0 +1,25 @@
+import { createElement } from 'lwc';
+import WeatherComponent from 'c/weatherComponent';
+
+describe('c-weather-component', () => {
+    afterEach(() => {
```

Comment:
I don't think you are done here, but it is really great that you started providing UI testing. That is more than what was required.

Reply:
Hey, Daniel! This test file gets automatically created when creating a new LWC. Haha!
But yeah, I want to learn more about UI testing. Seems useful!